force-app/main/default/classes/extentionController.cls

```diff
@@ -0,0 +1,81 @@
+public class extentionController {
```

Comment:
Class names have to start from Capital Letter.
it is OK for now, but for future please use capital letter